src/index.html

```diff
@@ -17,6 +17,12 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="stopwatch__container">
+        <div class="stopwatch__center"></div>
+        <div class="stopwatch__mimutes stopwatch__mimutes--speed-up"></div>
```

Comment:
There is a typo in the class name `stopwatch__mimutes`. It should likely be `stopwatch__minutes`. This typo could cause issues if the CSS or JavaScript relies on this class name.